src/app/features/dashboard/pagination/pagination.component.spec.ts

```diff
@@ -1,9 +1,9 @@
-import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { Store } from '@ngxs/store';
```

Comment:
@KarelDiaz remember that when you import dependencies should be ordered by 3 classifications:
```
//angular dependencies
import { async, ComponentFixture, TestBed } from '@angular/core/testing';
import { NO_ERRORS_SCHEMA } from '@angular/core';

// 3r parties dependencies
import { Store } from '@ngxs/store';

// project dependencies
import { PaginationComponent } from './pagination.component';
import { NextPageAction, PreviousPageAction } from '../../../core/redux/actions';
```